sdk/onlineexperimentation/arm-onlineexperimentation/test/public/onlineExperimentation_operations_test.spec.ts

```diff
@@ -36,7 +36,7 @@ describe("OnlineExperimentation test", () => {
   afterEach(async function () {
     await recorder.stop();
   });
-  it("operations list test", async function () {
+  it.skip("operations list test", async function () {
```

Comment:
@smichtch
FYI the apiversion 2025-08-01-preview doesn't support operation_list api

error message:
```
"The resource type 'Operations' could not be found in the namespace 'Microsoft.OnlineExperimentation' for api version '2025-08-01-preview'. The supported api-versions are '2025-05-31-preview'."
```

Reply:
@kazrael2119 please use commit: 64a54347ebef32101bd1fcd29d23476b4b838bba in tsp-location.yml as noted in the issue. We just want to include a model rename in this release, not the new api-version yet.

Reply:
have re-gened